pkg/bpmn_engine/engine_timer_test.go

```diff
@@ -27,6 +27,24 @@ func TestEventBasedGatewaySelectsPathWhereTimerOccurs(t *testing.T) {
 	then.AssertThat(t, cp.CallPath, is.EqualTo("task-for-message"))
 }
 
+func TestInvalidTimer(t *testing.T) {
```

Comment:
Just a slight renaming, to express the intent

```suggestion
func TestInvalidTimer_will_stop_continue_execution(t *testing.T) {
```